### DISCUSSION TITLE
Add the ability to navigate from the comment to the commented commit

### DISCUSSION THREAD

**Discussion by jacek-lewandowski** [community/community#44751] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

There is a pull request with many commits. It was reviewed and the comments were added when the reviewer was viewing changes in a commit by commit manner rather than on full changeset. Now all the comments appears on the primary pull request page, however I cannot navigate from the comment back to the commit on which the comment was made. In order to figure out which commit was commented, I need to go through all the commits and see the comments there. 

It would be great if the tooling around reviewing commit by commit was extended, in particular I'd find it very useful if:
- can navigate from the comment to the commented commit
- the commits having comments are somehow distinguished
- comments have something viewed state that can be changed when applying remarks
- can see comments filtered by viewed state

Thanks,
Jacek

**Comment by soroshsabz** [comment]
I think it is very useful feature
